docs/docs/guides/multi-stage-application-delivery.md

```diff
@@ -0,0 +1,340 @@
+---
+comments: true
+---
+
+# Multi-stage Application Delivery
+
+In this guide you will learn how to deploy
+an application across multiple stages while getting
+full observability of each of your workload deployments.
+This is shown based on an example where we deploy
+an application into two environments (`dev` and `production`),
+using [Argo CD](https://argo-cd.readthedocs.io/en/stable/).
+
+The promotion from `dev` to `production` is implemented using
+`promotionTasks` that can be executed after a successful
+application deployment.
+In this example, the promotion task creates a pull request
+to deploy the same workload version that has been deployed in the
+`dev` stage into the `production` stage.
+
+Using the observability capabilities of Keptn,
+we also add contextual metadata to each deployment,
+such as the commit ID which triggered the deployment of a new
+version.
+This metadata is added to the OpenTelemetry traces
+representing the deployment of an application version
+in a particular stage.
+Also, deployment traces for the `production` stage will
+contain references to the deployments of the `dev` stage,
+giving you full traceability of a workload version
+across different stages.
+
+## Before you begin
+
+This guide assumes the following:
```

Comment:
I would like this list to be imperative with a parallel structure for all items (Install Keptn, Install ArgoCD, Set up observability, etc.) but I see that the remaining items don't work so well for an imperative list -- you need the repo but may already have it.  Maybe you could break the list up into two lists -- "Do the following:" and "You also need:"?  Or leave it as it is.